tests/test_updater.py

```
@@ -5,10 +5,9 @@
 
 import pytest
 
-import fakes
+from .fakes import FakeFiles, FakeSpreadsheets, FakeWeb
 import mlbstandings.updater
```

Pylint (3.3.7)

[error] Unable to import 'mlbstandings.updater'

(E0401)